exodus_core/analysis/test_exodus_analyze.py

```diff
@@ -73,8 +73,8 @@ def test_icon_similarity_different_app(self):
         diff_1 = sa.get_icon_similarity(phash_1, phash_2)
         diff_2 = sa.get_icon_similarity(phash_2, phash_1)
 
-        self.assertEqual(diff_1, 0.7265625)
-        self.assertEqual(diff_2, 0.7265625)
+        self.assertEqual(diff_1, 0.734375)
+        self.assertEqual(diff_2, 0.734375)
```

Comment:
If this works, I would need @U039b to confirm whether this is something we can accept or not.

Reply:
It looks good to me even if it is strange that upgrading pillow has a limited impact on perceptual hashes.